src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
```

Comment:
You can use enum in your figure now
```suggestion
  shape: Shapes;
```